lib/mastodon/rest/request.rb

```diff
@@ -19,6 +19,8 @@ def initialize(client, request_method, path, options = {})
       def perform
         options_key = @request_method == :get ? :params : :form
         response    = http_client.headers(@headers).public_send(@request_method, @uri.to_s, options_key => @options)
+        STDERR.puts response.body if ENV['DEBUG'] == 'true'
```

Comment:
Hmm there is a wonderful gem called httplog which can be set to log outgoing requests and responses, you could use that for debugging.